docs/src/main/asciidoc/spring-cloud-sleuth.adoc

```diff
@@ -446,9 +446,10 @@ String countryCode = ExtraFieldPropagation.get(span.context(), "x-country-code")
 ```
 
 IMPORTANT: A difference from previous versions of Sleuth is that, with Brave, you must pass the list of baggage keys.
-There are two properties to achieve this.
+There are the following properties to achieve this.
 With the `spring.sleuth.baggage-keys`, you set keys that get prefixed with `baggage-` for HTTP calls and `baggage_` for messaging.
-You can also use the `spring.sleuth.propagation-keys` property to pass a list of prefixed keys that are whitelisted without any prefix.
+You can also use the `spring.sleuth.propagation-keys` property to pass a list of prefixed keys that are propagated without any prefix.
+You can also use the `spring.sleuth.redacted-keys` property to pass a list keys that will be available locally but will not be propagated over the wire.
```

Comment:
should we call this local-keys? cc @shakuzen @devinsba @jeqo

Reply:
if we call this local-keys, we should clarify "will be available locally" as "will be propagated locally"

Reply:
Yea, I would prefer terminology other than redacted, since it is but one use case and implies perhaps more than this is (e.g. making something available in logging that's "redacted" seems off to me). `local-keys` as suggested sounds good to me.
While we're polishing the documentation here, perhaps the description for `propagation-keys` could be updated to something like "keys that are propagated to remote services without any prefix".

Reply:
we can also change brave with deprecation (we'd never be able to remove that method before a major, but at least it could clarify)